doc/faq.md

```diff
@@ -102,7 +102,7 @@ We recommend to lookup parameters of your projection (like the validity extent)
 over at [epsg.io](https://epsg.io/).
 
 
-## Why is my map centered on the gulf of guinea (or africa, the ocean, null-island)?
+## Why is my map centered on the gulf of guinea (or Africa, the ocean, null-island)?
```

Comment:
=> ?
## Why is my map centered on the Gulf of Guinea (or Africa, the ocean, null-island)?